proto/src/serialize/binary/restrict.rs

```diff
@@ -0,0 +1,131 @@
+/// Untrusted types are boxed in this, only access is through the verify method
```

Comment:
Nit: I'm not sure if it's accurate to say "boxed" here. To me, "boxed" implies that data is allocated on the heap. It might be more accurate to say "wrapped". Admittedly, this is minor.

Reply:
I agree with @hawkw. Boxed is reserved term in Rust. Wrapped is better.

Reply:
Changed.